versions/p-/paraview.json

```
@@ -1,8 +1,13 @@
 {
   "versions": [
+    {
+      "git-tree": "81c9cf7521c4492a7ba5719d4d7e46a34d965dc9",
+      "version-semver": "5.10.1",
+      "port-version": 0
+    },
     {
       "git-tree": "8f5d22c6754f293f54a360472f78e760f06aae80",
-      "version": "5.9.1",
+      "version-semver": "5.9.1",
```

```suggestion
      "version": "5.9.1",
```
We should only add content to the record files.